features/step_definitions/common_stepdef.rb

```diff
@@ -158,14 +143,37 @@
   end
 end
 
-Then(/^I create a new procurement/) do
-  step 'I am a logged in user - buildings account'
-  step 'I am on your account page'
-  step 'I click on start a procurement'
-  step 'I am on "What happens next" page'
-  step 'I click on "Continue"'
-  step 'I am on "Contract name" page'
-  step 'I add contract name'
-  step 'I click on "Save and continue"'
-  step 'I am on "Requirements" page'
+And(/^I should see error message header "([^"]*)"$/) do |heading|
+  expect(common.header_two.text).to eq(heading)
+end
+
+And(/^I should see error message header "([^"]*)"$/) do |heading|
+  expect(common.header_two.text).to eq(heading)
+end
+
+# And(/^The following text is displayed:$/) do |table|
+#   table.transpose.raw.flatten.each do |item|
+#     expect(page).to have_css('div h2', text: item)
+#   end
+# end
+Then("I should see the following selected services heading") do |table|
+  expect(page).('#css-list-basket > div > h2 > span:nth-child(2)').to eq(table)} 
+end
+
+
+=begin
+Given("I click on {string} checkbox") do |string|
+  check ('child law')
+end
+=end
```

Comment:
For multiline comments, it is more common just to use the `#` to comment out each line individually.
```suggestion
# Given("I click on {string} checkbox") do |string|
#   check ('child law')
# end
```